<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>209</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="dirLists">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addDir">
         <property name="text">
          <string>Dodaj katalog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rmDir">
         <property name="text">
          <string>Usuń katalog</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>241</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="imageLists">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QListView" name="list_pictures"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Manhatan</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Szukaj</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="list_similar"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>381</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="imagePreviews">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QGraphicsView" name="image_preview"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="similarPreview"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionBuduj_kolekcj"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="actionBuduj_kolekcj">
   <property name="text">
    <string>Buduj kolekcję</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBuduj_kolekcj</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>createCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rmDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addDirectory()</slot>
  <slot>removeDirectory()</slot>
  <slot>createCollection()</slot>
 </slots>
</ui>
